src/foam/box/BoxRegistryBox.js

```diff
@@ -45,7 +45,7 @@ foam.CLASS({
         if ( this.SubBoxMessage.isInstance(msg.object) ) {
           var name = msg.object.name;
 
-          if ( this.registry[name].localBox ) {
+          if ( this.registry[name] && this.registry[name].localBox ) {
```

Comment:
Previous check was insufficient. If `this.registry[name]` is not defined, it will not move on to attempting to send an error back through `msg.attributes.errorBox`